code/src/components/app.css

```diff
@@ -0,0 +1,12 @@
+* {
+    box-sizing: border-box;
+    font-family: 'Trispace', sans-serif;
+}
+
+.app__grid {
+    background: #fff;
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
```

Comment:
Not sure why there are three columns. When looking at the page the content is not centered, so maybe one column would be enough and then add rows for every thought post